client/lib/cart-values/cart-items.js

```diff
@@ -647,6 +653,22 @@ function getIncludedDomain( cartItem ) {
 	return cartItem.extra && cartItem.extra.includedDomain;
 }
 
+function setVolume( cartItem, volume ) {
+	function setItemVolume( items ) {
+		return items.map( item => {
+			if ( item.meta === cartItem.meta && ( item.product_id === cartItem.product_id || isPrivacyProduct( item ) ) ) {
+				return extend( {}, item, { volume } );
+			}
+
+			return item;
+		} );
+	}
+
+	return function( cart ) {
+		return React.addons.update( cart, { products: { $apply: setItemVolume } } );
+	};
+}
```

Comment:
That won't work because spread operator only works with Arrays, iterables in ES6. JSX uses it but it's limited to ES6. We might expect to use the operator over objects in ES7.